models/spring-ai-zhipuai/src/main/java/org/springframework/ai/zhipuai/api/ZhiPuAiApi.java

```diff
@@ -99,15 +103,20 @@ public ZhiPuAiApi(String baseUrl, String zhiPuAiToken, RestClient.Builder restCl
 	 */
 	public ZhiPuAiApi(String baseUrl, String zhiPuAiToken, RestClient.Builder restClientBuilder, ResponseErrorHandler responseErrorHandler) {
 
+		Consumer<HttpHeaders> authHeaders = h -> {
+			h.setBearerAuth(zhiPuAiToken);
+			h.setContentType(MediaType.APPLICATION_JSON);
```

Comment:
i'll run the IT to make sure adding content type is ok.

Reply:
unfortunately, my api key no longer works.  will have to follow up on it another time.